src/snowflake/connector/arrow_iterator.pyx

```diff
@@ -1,6 +1,4 @@
-#
-# Copyright (c) 2012-2019 Snowflake Computing Inc. All right reserved.
-#
+# Copyright (c) 2012-2020 Snowflake Computing Inc. All right reserved.
```

Comment:
Nit: Can we keep the three lines for the copyright with this tool?

Reply:
Sure, I didn't think this tool would be smart enough, but I tried and it worked 🎉
But that made the C++ comments 5 lines long, so I switched those to `//` style instead, hope that's okay with you!